src/lib/endpoints.ts

```diff
@@ -25,7 +24,7 @@ export const fetchVoteCastEventEndpoint = `/address/{{address}}/eventsByTopic0?t
 
 export const getNftInfo = process.env.NEXT_PUBLIC_API_RWS_NFT_INFO || `/nfts/{{nftAddress}}?${CHAIN_ID_PARAM}`
 
-const NEW_ALLOCATION_EVENT = ethers.id('NewAllocation(address,address,uint256)')
+const NEW_ALLOCATION_EVENT = '0xed07ca57097393e77ba36105a07f6810afc2180b72a9c02a4b0da4b51a73a6ec'
```

Comment:
please clarify where this comes from e.g.: keccak256('NewAllocation(address,address,uint256)'